hw/syn/data/common_syn_cfg.hjson

```diff
@@ -0,0 +1,28 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  project:          opentitan
+  doc_server:       docs.opentitan.org
+  results_server:   reports.opentitan.org
+
+  flow:             synthesis
```

Comment:
Would prefer to use one of {synthesis|syn} and stick to it everywhere (folder name, flow name, mk file name etc).

Reply:
agreed. I've used syn everywhere else since we tend to use abbreviated names for everything. `sim`, `lint`, `dv`, `fpv`, ...